conans/test/functional/toolchains/cmake/cmakedeps/test_apple_frameworks.py

```diff
@@ -503,7 +503,7 @@ def test(self):
 
 @pytest.mark.tool("cmake")
 @pytest.mark.skipif(platform.system() != "Darwin", reason="Only OSX")
-def test_m1():
+def test_iphoneos_crossbuild():
```

Comment:
Renamed because this particular test does _not_ test any logic that appears directly related to M1 or Apple Silicon, but rather a simple case of cross-compiling for iOS.